.github/workflows/review_app_creation.yml

```
@@ -0,0 +1,174 @@
+name: 🕵 Création d'une recette jetable
+
+# L'action est lancée quand une étiquette est ajoutée à la PR ou qu'un push est effectué sur la branche.
+# `types: [ synchronize ]` cible les événements `push`.
+on:
+  pull_request:
+    types: [ labeled, synchronize ]
+  # https://docs.github.com/en/actions/how-tos/manage-workflow-runs/manually-run-a-workflow
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref_name }}-${{ github.event.pull_request.number || github.sha }}
+
+env:
+  SUPABASE_ACCESS_TOKEN: ${{ secrets.TEST_SUPABASE_ACCESS_TOKEN }}
+  SUPABASE_PROJECT_ID: ${{ secrets.TEST_SUPABASE_PROJECT_ID }}
+  BRANCH: ${{ github.head_ref }}
+  PR_ID: ${{ github.event.number }}
+  SCALINGO_DJANGO_APP_NAME: docurba-django-demo
+  SCALINGO_NUXT_APP_NAME: docurba-nuxt-demo
+  NUXT3_REPO: git@github.com:betagouv/docurba-nuxt3.git
+  NUXT3_REPO_HTTPS: https://github.com/betagouv/docurba-nuxt3/
+  NUXT3_BRANCH: main
+
+jobs:
+  create:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    if: github.event.action == 'labeled' && github.event.label.name == 'recette-jetable'
+
+    steps:
+    - name: 🚶‍♂️‍➡️ Déplacement vers la branche principale
+      uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+      with:
+        ref: ${{ github.head_ref }}
+
+    - name: 📥 Récupération des branches Git
+      run: git fetch --prune --unshallow
+
+    - name: 🏷️ Définition des noms de la recette jetable
+      run: |
+        # Le nom est déterminé automatiquement par Scalingo pour les applications Nuxt2 et Django
+        # car ce sont des recettes jetables.
+        # Suivons le même modèle pour Supabase et Nuxt3.
+        echo "SUPABASE_REVIEW_APP_NAME=supabase-pr${PR_ID}" >> $GITHUB_ENV
+        echo "NUXT_3_REVIEW_APP_NAME=nuxt3-pr${PR_ID}" >> $GITHUB_ENV
```

Préfixons par `docurba-`
```suggestion
        echo "NUXT_3_REVIEW_APP_NAME=docurba-nuxt3-pr${PR_ID}" >> $GITHUB_ENV
```